Update _index.md (#37)
FSHSchool/site: content/docs/Tutorials/basic/_index.md

║ ╰──────────┴────────────┴───────────┴─────────────┴───────────╯ ║
║                                                                 ║
╠═════════════════════════════════════════════════════════════════╣

> ║ Nice! You're totally killing it!      0 Errors      0 Warnings ║

Reviewer: ```suggestion
║ You are dolphinitely doing great!      0 Errors      0 Warnings ║
```
Since we always use a pun, I'd like to swap out the inappropriate one with this one instead.  (Sorry -- we did not realize "krilling" had a slang meaning!).
